pom.xml

```diff
@@ -108,6 +108,11 @@
       <artifactId>domain-models-runtime</artifactId>
       <version>${rmb.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-di-support</artifactId>
+      <version>1.0.0</version>
+    </dependency>
```

Comment:
@dmtkachenko -- curious as to why this is added dependency is required. Would like to avoid added complexity on this module as much as possible.

Reply:
we'll benefit from it once RM API libs is in place. then we'll be able to refactor the code and use Spring DI and conversion service (as we did in mod-kb-ebsco-java). mod-codex-ekb is very similar to mod-kb-ebsco-java so it's logically to apply the same approach

Reply:
Ok - thanks, understood on similarities between modules. I'm concerned over follow on work which would introduce Spring DI into this module and to what extent the module will be refactored to support this. It's a relatively simple module and adding more dependencies could make it more challenging for a developer to  maintain. Is it possible to create/use a common library/services and not use DI?

Reply:
well, to me Spring DI actually simplify the code by moving class instantiation part out to the framework. you don't need to worry about object initialization, you just declare that your class needs this, this and this. it also helps write unit tests because class' dependencies can be mocked easily. 
but we can think about other ways